### PR TITLE
bpo-36345: Add a new example in the documentation of wsgiref

### PR DESCRIPTION
**Pull request by matrixise**
It's the second PR where I convert the code of `Tools/scripts/serve.py` as an example in the documentation of `wsgiref`.

https://bugs.python.org/issue36345

### CODE REVIEW

**Review by eamanu — APPROVED** [review]
lgtm. IMO should be good tell how the example is executed.

File: Doc/library/wsgiref.rst [hunk 1]
```diff
@@ -781,3 +781,35 @@ This is a working "Hello World" WSGI application::
 
        # Serve until process is killed
        httpd.serve_forever()
+
+
+Example of a small wsgiref based web server::
```
Comment (brettcannon) [inline Doc/library/wsgiref.rst]:
```suggestion
Example of a small wsgiref-based web server::
```

**Review by brettcannon — APPROVED** [review]

**Comment by brettcannon** [comment]
One minor change and then we can merge this! (I tried to merge it myself but you didn't give repo owners edit rights on your PRs, @matrixise ).

**Comment by matrixise** [comment]
Thank you so much @brettcannon , About the rights, this is strange because I just create this PR and I think the flag was checked. really sorry.

**Comment by brettcannon** [comment]
Thanks!

**Pull request merged**